packages/test/test-utils/src/timeoutUtils.ts

```diff
@@ -25,9 +26,17 @@ export async function timeoutAwait<T = void>(
     return Promise.race([promise, timeoutPromise<T>(() => { }, timeoutOptions)]);
 }
 
-export async function ensureContainerConnected(container: Container): Promise<void> {
-    if (!container.connected) {
-        return timeoutPromise((resolve) => container.once("connected", () => resolve()));
+export async function ensureContainerConnected(
```

Comment:
There is another function `waitForContainerConnection` in `TestSummaryUtils.ts` that does this but without the timeout. Maybe consolidate that too?

Reply:
Thanks for pointing it out! I'll move everything to the new function.